components/feature/downloads/src/test/java/mozilla/components/feature/downloads/SimpleDownloadDialogFragmentTest.kt

```diff
@@ -45,22 +51,59 @@ class SimpleDownloadDialogFragmentTest {
             isOnStartDownloadCalled = true
         }
 
-        dialog.onStartDownload = onStartDownload
-        dialog.testingContext = testContext
+        val fragment = Mockito.spy(SimpleDownloadDialogFragment.newInstance())
 
-        performClick(BUTTON_POSITIVE)
+        fragment.onStartDownload = onStartDownload
+        fragment.testingContext = testContext
+
+        Mockito.doReturn(testContext).`when`(fragment).requireContext()
```

Comment:
nit: we usually import statically to avoid `Mockito.doReturn`...

Reply:
Done!